local.tf

```diff
@@ -70,6 +70,15 @@ resource "random_pet" "domain_name" {
   }
 }
 
+resource "random_pet" "msi_name" {
+  prefix    = "msi-aks"
```

Comment:
I think this goes a bit into the wrong direction. We are using _random_pet_ in `locals.tf` 8 times. This gives us 8 different random pets. IMO we should only generate a single random pet resource and combine it with various suffixes. This way the random pet name can be used across resources to identify resources belonging to the same setup.

Reply:
These 8 were my doing. We can coalesce on to a single random string and prefix where appropriate. I'll open a small PR in a minute to address

Reply:
Variable tidy up PR here #11

Reply:
Any chance we can review this PR so I can add more features for aks cluster?

Reply:
Done! See additional comments